src/valhalla_benchmark_loki.cc

```diff
@@ -119,37 +124,53 @@ bool ParseArguments(int argc, char* argv[]) {
   } catch (std::exception& e) {
     std::cerr << "Unable to parse command line options because: " << e.what() << "\n"
               << "This is a bug, please report it at " PACKAGE_BUGREPORT << "\n";
-    return false;
+    return 1;
   }
 
   if (vm.count("help")) {
     std::cout << options << "\n";
-    return true;
+    return -1;
   }
 
   if (vm.count("version")) {
     std::cout << "loki_benchmark " << VALHALLA_VERSION << "\n";
-    return true;
+    return 0;
   }
 
   // argument checking and verification
   for (const auto& arg : std::vector<std::string>{"config", "input_files"}) {
     if (vm.count(arg) == 0) {
       std::cerr << "The <" << arg << "> argument was not provided, but is mandatory\n\n";
       std::cerr << options << "\n";
-      return false;
+      return 1;
     }
   }
 
   // TODO: complain when no input files
 
-  return true;
+  return 0;
+}
+
+valhalla::sif::cost_ptr_t create_costing() {
```

Comment:
we expand the benchmark to allow for you to select other costings than the `none` costing